lib/generators/templates/rpush.rb

```diff
@@ -42,10 +42,7 @@
   # Called when a notification is queued internally for delivery.
   # The internal queue for each app runner can be inspected:
```

Comment:
Can't say I've used this before, but since the comment says "each app runner", is there currently no way to inspect all of them? If not, what does `.status` return? I'm unfamiliar with this API.

Reply:
`.status` provides the best approximation to the data that I believe this old form was exposing.

https://github.com/rpush/rpush/commit/f2a3c23f5b6644d602307f5f82789fcf9879f651 removed the implementation of this example.

Reply:
I understand. Should we adapt the comment then?

Reply:
In what way would you like to adapt it?

Reply:
Sorry, I kind of lost track of this PR. Looking at it again, I believe my question with regards that the new comment only gets the status from _one_ runner? Or does this return the status from all runners, still?

Since the comment says, get the status from all runners.

Reply:
It's still all runners :)